src/plugins/intel_cpu/tests/unit/snippets_transformations/x64/lowered/buffer_allocation.cpp

```diff
@@ -29,126 +29,33 @@
-        pipeline.register_pass<ov::snippets::lowered::pass::FuseLoops>();
-        pipeline.register_pass<ov::snippets::lowered::pass::SplitLoops>();
-        pipeline.register_pass<ov::intel_cpu::pass::InsertBrgemmCopyBuffers>();
-        pipeline.register_pass<ov::snippets::lowered::pass::InsertBuffers>();
-        pipeline.register_pass<ov::snippets::lowered::pass::InsertLoadStore>(m_vector_size);
-        pipeline.register_pass<ov::snippets::lowered::pass::InitLoops>();
-        pipeline.register_pass<ov::snippets::lowered::pass::InsertLoops>();
-        pipeline.register_pass<ov::snippets::lowered::pass::AllocateBuffers>(m_is_buffer_optimized);
-        pipeline.run(m_linear_ir);
     }
 
-    void Validate() {
-        std::set<size_t> reg_groups, clusters;
-        for (const auto& buffer : m_linear_ir.get_buffers()) {
-            reg_groups.insert(buffer->get_reg_group());
-            clusters.insert(buffer->get_cluster_id());
-        }
-        EXPECT_EQ(reg_groups.size(), m_expected_reg_group_count);
-        EXPECT_EQ(clusters.size(), m_expected_cluster_count);
-        EXPECT_EQ(m_linear_ir.get_static_buffer_scratchpad_size(), m_expected_size);
+    void AddBackendSpecificPostSplitPasses(ov::snippets::lowered::pass::PassPipeline& pipeline) override {
+        pipeline.register_pass<ov::intel_cpu::pass::InsertBrgemmCopyBuffers>();
     }
 
-    virtual std::shared_ptr<ov::Model> GetModel(const std::vector<ov::PartialShape>& shapes) const = 0;
-
-    void MarkOp(const std::shared_ptr<ov::Node>& node,
-                const std::vector<std::vector<size_t>>& in_subtensors,
-                const std::vector<std::vector<size_t>>& out_subtensors) const {
-        OPENVINO_ASSERT(in_subtensors.size() == node->inputs().size(), "Incorrect count of input subtensors");
-        OPENVINO_ASSERT(out_subtensors.size() == node->outputs().size(), "Incorrect count of output subtensors");
-        // Mark input and output ports with the first supported subtensor
-        for (size_t i = 0; i < node->inputs().size(); ++i) {
-            const auto& input = node->input(i);
-            ov::snippets::lowered::PortDescriptorUtils::set_port_descriptor_ptr(
-                input,
-                std::make_shared<ov::snippets::lowered::PortDescriptor>(input, in_subtensors[i]));
-        }
-        for (size_t i = 0; i < node->outputs().size(); ++i) {
-            const auto& output = node->output(i);
-            ov::snippets::lowered::PortDescriptorUtils::set_port_descriptor_ptr(
-                output,
-                std::make_shared<ov::snippets::lowered::PortDescriptor>(output, out_subtensors[i]));
-        }
+    static void MarkOp(const std::shared_ptr<ov::Node>& node,
+                       const std::vector<std::vector<size_t>>& in_subtensors,
+                       const std::vector<std::vector<size_t>>& out_subtensors) {
+        BufferAllocationTest::MarkOp(node, in_subtensors, out_subtensors);
```

Comment:
Can we make `BufferAllocationTest::MarkOp` non-static to avoid duplication here?